### PR TITLE
User meeting scripts

### PR DESCRIPTION
**Pull request by jburel**
Scripts used for the user meeting
* Import data for other users by the importer user
* Use -T option instead of dataset

The`` import_as`` is probably better that the original script
like that all the users do not need to have the same password

cc @pwalczysko

### CODE REVIEW

File: maintenance/in_place_import_as.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# ------------------------------------------------------------------------------
+
+# This script imports in-place data for 40 different users by default, 
+# user-1 through user-40 into a target dataset.
+# The data is imported by a dedicated user with restricted admin
+# privileges on behalf of other users 
+# i.e. after import each of the 40 users has their own batch of data.
+# Data can also be imported for trainers.
+# To import for trainers run for example
+# OMEUSER=trainer NUMBER=2 bash in_place_import.sh
```
Comment (pwalczysko) [inline maintenance/in_place_import_as.sh]: ``in_place_import.sh``-> ``in_place_import_as.sh``

File: maintenance/in_place_import_as.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+IMPORTER=${IMPORTER:-importer}
+OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
+PASSWORD=${PASSWORD:-ome}
+HOST=${HOST:-outreach.openmicroscopy.org}
+FOLDER=${FOLDER:-siRNAi-HeLa}
+NUMBER=${NUMBER:-40}
+OMEUSER=${OMEUSER:-user}
+for ((i=1;i<=$NUMBER;i++));
+do  $OMEROPATH login --sudo ${IMPORTER} -u $OMEUSER-$i -s $HOST -w $PASSWORD
+    DatasetId=$($OMEROPATH obj new Dataset name=$FOLDER)
```
[the remark this answers is not in the document]
Comment (pwalczysko) [reply maintenance/in_place_import_as.sh]: Maybe this is a variation on https://trello.com/c/kWgFImSC/43-bug-import-target ? @jburel , I do not know the exact reason right now.

**Comment by pwalczysko** [comment]
I guess we need to figure out the ``-T`` logic carefully, I remember the script had originally ``-T`` but because of https://trello.com/c/kWgFImSC/43-bug-import-target I changed it.

**Comment by jburel** [comment]
@pwalczysko I will roll back

**Comment by jburel** [comment]
@pwalczysko will you be to retest tomorrow?

**Comment by pwalczysko** [comment]
The import as script https://github.com/ome/training-scripts/pull/8/files#diff-a5509e8a661374f9352596f334c3bb46 works fine now.

**Pull request merged**